src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -89,6 +89,14 @@ public virtual async Task<DValue<RecordValue>> AppendAsync(RecordValue record)
             throw new System.NotImplementedException("It is not possible to append to a TableValue directly.");
         }
 
+        // Return boolean value
+        // Async because derived classes may back this with a network call. 
+        public virtual async Task<DValue<BooleanValue>> RemoveAsync(RecordValue record)
```

Comment:
>RemoveAsync [](http://example.com/codeflow?start=56&length=11)

Why remove if we are implementing an update?

Reply:
Patch shouldn't call Remove.  But ok to add this for the upcoming Remove function